scripts/testhubot.coffee

```diff
@@ -182,6 +195,46 @@ module.exports = (robot) ->
                 " 151~ 매우나쁨  장시간 또는 무리한 실외 활동제한, 목의 통증과 기침등의 증상이 있는 사람은 실외활동을 피해야 함"
     return
 
+  readJSONFile = (callback) ->
+    jf.readFile file, (err, data) ->
+      if err
+        callback err, null
+      else
+        callback null, data
+
+  writeJSONFile = (data) ->
+    jf.writeFile file, data, (err) ->
+      console.log err  if err
+
+  removeAlarmJob = (msg) ->
+    readJSONFile (err, data) ->
+      if err
+        console.log err
+      else
+        tempData = []
+        data.forEach (obj) ->
+          tempData.push obj  if obj.msg isnt msg
+        writeJSONFile tempData
+
+  # 알람초기화. data.json 파일의 내용을 모두 알람으로 등록
+  init = ->
+    readJSONFile (err, data) ->
+      if err
+        console.log err
+      else
+        data.forEach (obj) ->
+          beforeMin = 30
+          cronDate = new Date(obj.time)
+          cronDate.setMinutes cronDate.getMinutes() - beforeMin
+          CronJob = require("cron").CronJob

```

Comment:
Dear,
It's not an issue with coding style :)
I think it's related with optimization and code readability.
Why do you repeatedly set the same value to beforeMin?
Why do you repeatedly require the same class CronJob?

But I will merge and modify by myself for your favor.
Thanks.